content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,110 @@
+
+Your noisiest logging service contains several logs, some of which may be irrelevant for troubleshooting. By identifying your noisiest logging service, you can quickly find the information you need to filter irrelevant logs by status.
+
+**To identify your noisiest logging service and status**:
+
+1. In Log Explorer, select graph view located next to the search bar. 
+2. Below the search bar, set `count` * group by `service` and `limit` to top 10.
+3. Select `Top List` from the dropdown menu next to `hide controls`.
+4. Click on the first listed service and select `search for` in the populated menu. This generates a search, which is visible in the search bar above, based on your `service` facet.
+5. Switch group by `service` to group by `status`. This generates a top statuses list for your service.
+6. Click on the first listed status and select `search for` in the populated menu. This adds your `status` facet to the search.
+
+**Note**: These steps are applicable to any high volume logging query to generate a top list. You can group by any facet, such as `host` or `client name` versus `service` or `status`.
+
+## 2. Identify noisy logs with log patterns
+Switch to the [patterns][2] view, located next to the graph view, to view the patterns associated with your service and status.
+
+{{< img src="logs/guide/log-patterns.png" alt="Log Patterns View" responsive="true" style="width:75%;">}}
+
+The patterns view is helpful when identifying and filtering noisy patterns. It shows the number of logs matching a pattern, split by service and status. Click on the first pattern to view a detailed log of events relating to your status. A contextual panel will populate with information about your noisiest status pattern.
+
+## 3. Set an exclusion filter for noisy logs
+
+The selected pattern contextual panel lists every instance (event) of a log pattern and creates a custom parsing rule based on the selected pattern. This parsing rule is required when creating an [exclusion filter][3]. Exclusion filters remove noisy logs from your Log Explorer, but these logs still have the ability to be tracked with custom metrics.
+
+**To create an exclusion filter for your top log pattern**:
+
+1. Click on your top pattern from the pattern view list.
+2. Click the parsing rule button in the top right corner to automatically generate a parsing rule associated with this pattern.
+3. Copy the parsing rule and navigate to the Configuration page under Logs in the main left hand navigation.
+4. Select indexes and click on your associated index. This will populate the option to add an exclusion filter.
+5. Select “Add an Exclusion Filter”.
+6. Input the filter name, define the exclusion query by pasting the parsing rule you copied, and set an exclusion percentage.
+
+You can also generate the parsing rule by clicking the **View All** button in the pattern contextual panel and selecting the **</>** option next to the search bar.
```

Comment:
This is the preferred way to get the search query associated with a pattern working with an exclusion filter.